#
# This patch improves menu styling (based on version in 2.7.0
# which features a background pixmap for the menu). ---Mickey.
#
--- gtk+-2.6.10/gtk/gtkmenu.c	2005-08-18 16:10:58.000000000 +0200
+++ gtk+-2.7.0/gtk/gtkmenu.c	2005-04-07 21:56:57.000000000 +0200
@@ -2490,6 +2495,15 @@
 			   arrow_size, arrow_size);
 	}
     }
+  else if (event->window == menu->bin_window)
+    {
+      gtk_paint_box (widget->style,
+		     menu->bin_window,
+		     GTK_STATE_NORMAL,
+		     GTK_SHADOW_OUT,
+		     NULL, widget, "menu",
+		     -border_x, -border_y, width, height);
+    }
 }
 
 static gboolean
